grupo-gomez-lopez/src/components/sharedButton/sharedButtonCSS.js

```diff
@@ -0,0 +1,2 @@
+const sharedButtonCSS = { margin: '50px' };
```

Comment:
Same principle here, we shouldn't declare styles outside our css modules

Reply:
Done